go-controller/README.md

```diff
@@ -39,9 +39,15 @@ Usage:
   -cluster-subnets string
      cluster wide IP subnet to use (default: 11.11.0.0/16)
   -init-master string
-     initialize master (that watches pods/nodes/services/policies), requires the hostname as argument
+     initialize master which enables both cluster manager (allocates node subnets) and network controller manager (which watches pods/nodes/services/policies and creates OVN db resources), requires the hostname as argument
+  -init-cluster-manager string
+     initialize cluster manager that watches nodes (allocates subnet for each node from the cluster-subnets), requires the hostname as argument and doesn't connect to the OVN dbs.
+  -init-network-controller-manager string
+     initialize network-controller-manager (which watches pods/nodes/services/policies and create OVN db resources), requires the hostname as argument and doesn't connect to the OVN dbs.
```

Comment:
Last part has to be removed, the network-controller-manager do connect to OVN dbs

Reply:
Ack